src/main/java/org/maxandersen/starfish/Starfish.java

```diff
@@ -168,6 +169,16 @@ public static void editConfig()throws Exception{
 
 }
 
+//Function to Launch the Editor
+public static void launch_editor(Path directory,String ide,String final_clone_path)throws IOException, InterruptedException{
+//If OS is windows then we add .exe after the command    
+boolean isWindows =System.getProperty("os.name").toLowerCase().indexOf("windows")>=0;
```

Comment:
this is fine for now but this should come from the configuration as a editor command could also be named code.cmd or code.bat on windows thus just adding .exe is not always the right answer..